catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceSource.java

```diff
@@ -174,18 +184,22 @@ public boolean isAvailable() {
       isAvailable = lastAvailable;
     }
 
-    if (lastAvailable != isAvailable) {
+    updateMonitors(isAvailable);
+
+    lastAvailable = isAvailable;
```

Comment:
❗️ I see concurrency issues around `lastAvailable` and `lastAvailableDate` which can be accessed modified from many simultaneous threads. In addition because